src/index.js

```diff
@@ -1,8 +1,45 @@
 import React from 'react';
-import ReactDOM from 'react-dom';
+import { BrowserRouter, Routes, Route } from 'react-router-dom';
```

Comment:
If you're using React Router...

Take a look at `React.Lazy`, so you don't have to import all these components on the initial page load.